src/hooks/useCookie.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+import Cookie from 'js-cookie';
+
+const useCookie = () => {
```

Comment:
Not sure that we need this hook. It will be used only on login page...

Reply:
Fixed.